src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RemoteRazorProjectFileSystem.cs

```diff
@@ -89,6 +90,22 @@ protected override string NormalizeAndEnsureValidPath(string path)
 
             absolutePath = _filePathNormalizer.Normalize(absolutePath);
             return absolutePath;
+
+            static bool IsPathRootedForPlatform(string path)
+            {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && path == "/")
+                {
+                    // We have to special case windows and "/" because for some reason Path.IsPathRooted returns true on windows for a single "/" path.
```

Comment:
This is slightly concerning because `Path.IsPathRooted` is supposed to be the source of truth. Do we know why they are returning `true` for `/` on Windows?  Should we file a bug or do they have a reason to do that?

Reply:
I thought it was odd as well but when I looked at the source of the method it stated:

```
        // Tests if the given path contains a root. A path is considered rooted
        // if it starts with a backslash ("\") or a drive letter and a colon (":").
```

And the impl specifically allows the alternate directory separator character (`/`). Didn't my question of "why" but it was explicit.